src/main/kotlin/no/nav/familie/ef/iverksett/behandlingstatistikk/BehandlingstatistikkRepository.kt

```diff
@@ -0,0 +1,39 @@
+package no.nav.familie.ef.iverksett.behandlingstatistikk
+
+import no.nav.familie.ef.iverksett.util.queryForJson
+import no.nav.familie.eksterne.kontrakter.saksstatistikk.ef.BehandlingDVH
+import no.nav.familie.kontrakter.ef.iverksett.Hendelse
+import no.nav.familie.kontrakter.felles.objectMapper
+import org.springframework.jdbc.core.namedparam.MapSqlParameterSource
+import org.springframework.jdbc.core.namedparam.NamedParameterJdbcTemplate
+import org.springframework.stereotype.Repository
+import java.util.UUID
+
+@Repository
+class BehandlingstatistikkRepository(private val namedParameterJdbcTemplate: NamedParameterJdbcTemplate) {
+
+    fun lagre(behandlingId: UUID, behandlingDVH: BehandlingDVH, hendelse: Hendelse) {
+        return lagreBehandlingstatistikk(behandlingId, behandlingDVH, hendelse)
+    }
+
+    fun hent(behandlingId: UUID, hendelse: Hendelse): BehandlingDVH? {
+        val sql = "SELECT behandling_dvh FROM behandling_statistikk WHERE behandling_id = :behandlingId AND hendelse= :hendelse "
+        val mapSqlParameterSource = MapSqlParameterSource("behandlingId", behandlingId)
+        mapSqlParameterSource.addValue("hendelse", hendelse.toString())
+        return namedParameterJdbcTemplate.queryForJson(sql, mapSqlParameterSource)
```

Comment:
Hvis denne hadde vært `?: throw... Finner ikke hendelse=X for behandling=Y` så kan man sette att denne returnerer `BehandlingDVH` uten `?` sånn att de metodene som kaller på denne ikke trenger å bruke `!!`

Reply:
Ja, det går an

Reply:
Ja, det går an